docs/member/fixed-income.md

```diff
@@ -0,0 +1,128 @@
+If you are just entering the world of **investment, bonds, NCDs & MLDs** can be a great place to begin!
+So let’s understand what are Bonds, NCDs & MLDs and how investing in these securities can be beneficial for you?
+
+BONDS:
+A bond is a financial instrument that allows individuals to lend money to institutions such as governments or businesses. The institution will pay a set interest rate on the investment for the duration of the bond, then return the original amount at the end of the loan's term.
+Bond investing provides a predictable income stream, and in many cases, bonds pay interest multiple times in a year. Since the investor receives the entire principal amount if the bond is held until maturity, these are regarded as the best way to protect one's capital.
+
+NCDs:
+NCDs are Non-convertible Debentures. Companies use non-convertible debentures as a tool to raise long-term capital through a public issue. When compared to convertible debentures, lenders typically receive a higher rate of return to compensate for the disadvantage of non-convertibility.
+NCDs also provide the owner with a number of additional advantages, including increased liquidity due to their listing on the stock market, source tax exemptions, and safety due to the fact that only companies with strong credit ratings are permitted to issue NCDs in accordance with RBI-established guidelines.
+
+MLDs:
+MLDs are Market Linked Debentures. MLDs are non-convertible debentures with variable returns that are correlated to the market. The performance of the underlying index affects the return from a Market Linked Debenture.
+
+Now let’s understand how you can begin trading on the AlgoBulls Fixed Income marketplace.
+
+## 2. An Overview:
+
+To begin using the Fixed Income marketplace, download the AlgoBulls App or register on [www.algobulls.com](http://www.algobulls.com)
+
+After signing in, go to the Fixed Income section in the sidebar navigation. You will see the following fields:
+
+**i. Marketplace:** Find all the bonds, NCD & MLD cards in this marketplace. You can Buy or Sell these securities, view Security Details, use the Investment Calculator & view the Payment Cycle.
+
+[![Fixed Income](imgs/fixed_income_1.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/bonds_marketplace_1fixed_income_1.png)
+
+**ii. Fixed Income Card:** Get complete details, such as the Bond & NCD name, price, rating, maturity date, yield per year % and more here. Click on the know more to view the expanded version.
+
+[![Fixed Income](imgs/fixed_income_card.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/fixed_income_card.png)
+
+**iii. Search Bar:** To find a specific Security, enter its name or ISIN number into the search bar.
```

Comment:
`enter its name`

change it to

`enter the name`

Reply:
Done